packages/client/README.md

```diff
@@ -689,20 +766,35 @@ The library allows the user to choose a _partition key_, which simplifies publis
 The partition key can be given as an argument to the `publish` methods, and the library assigns a deterministic partition number automatically:
 
 ```js
-await client.publish('my-stream-id', msg, Date.now(), msg.vehicleId)
-
-// or, equivalently
+// msg.vehicleId being the partition key
+await streamr.publish(streamId, msg, Date.now(), msg.vehicleId)
+// or, equivalently, using the stream object
 await stream.publish(msg, Date.now(), msg.vehicleId)
 ```
+You can also specify the partition number as the last parameter:
+```js 
+await streamr.publish(streamId, msg, Date.now(), 4)
+// or, equivalently, using the stream object
+await stream.publish(msg, Date.now(), 4)
+```
+Alternatively, you can specify the partition number as part of the stream id:
+```js
+await streamr.publish({
+    id: `${address}/foo/bar`,
+    partition: 4
+}, msg, Date.now())
+```
 
-### Subscribing to partitioned streams
+#### Subscribing to partitioned streams
 
-By default, the JS client subscribes to the first partition (partition `0`) in a stream. The partition number can be explicitly given in the subscribe call:
+By default, the JS client subscribes to the first partition (partition `0`) in a stream. This behavior will change in the future so that it will subscribe to all partitions by default.
```

Comment:
```suggestion
By default, the client subscribes to the first partition (partition `0`) of a stream. Be aware: this behavior will change in the future so that it will subscribe to _all_ partitions by default.
```